Article/Article.js

```diff
@@ -3,24 +3,122 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.domElement = domElement;
+    // create a reference to the ".expandButton" class.
+    this.expandButton = domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = "Click to Expand";
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+
+    this.deleteButton = domElement.querySelector('.close');
+
+    this.expandButton.addEventListener('click', this.toggleArticle.bind(this));
+    this.deleteButton.addEventListener('click', this.deleteArticle.bind(this));
```

Comment:
You _could_ get around using `bind` by using arrow functions instead. For example:
```javascript
this.deleteButton.addEventListener('click', () => this.deleteArticle());
```